backend/internal/oauth/oauth2/granthandlers/init.go

```diff
@@ -37,6 +38,7 @@ func Initialize(
 	flowExecService flowexec.FlowExecServiceInterface,
 ) GrantHandlerProviderInterface {
 	authzService := authz.Initialize(mux, applicationService, jwtService, flowExecService)
-	grantHandlerProvider := newGrantHandlerProvider(jwtService, userService, authzService)
+	tokenBuilder, tokenValidator := tokenservice.Initialize(jwtService)
```

Comment:
Feels this should be injected from the root level init method. Same with authz service.
WDYT @senthalan ?